tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -15,11 +15,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from __future__ import annotations\n",
+    "\n",
     "import numpy as np\n",
     "import networkx as nx\n",
     "\n",
-    "from qiskit import Aer\n",
-    "from qiskit.algorithms import NumPyMinimumEigensolver"
+    "from qiskit.algorithms.minimum_eigensolvers import NumPyMinimumEigensolver"
```

Comment:
Should we move this down to where it says about using this otherwise when one starts reading about QAOA the first thing I see right here being imported from algorithms is something other than that.